examples/test_examples.py

```diff
@@ -98,3 +99,9 @@ def test_generation(self):
         with patch.object(sys, "argv", testargs + [model_type, model_name]):
             result = run_generation.main()
             self.assertGreaterEqual(len(result[0]), 10)
+
+    def test_bart_cnn_summarization(self):
```

Comment:
takes 15 seconds. Happy to mark it slow or move it somewhere else!

Reply:
yes let's maybe keep that in slow

Reply:
Non-trivial to import `slow` from tests.utils, should I reimplement, hack sys.path, move tests/utils to src/transformers or a better idea I haven't thought of?

Reply:
ha no just leave it as non-slow then